FlowCryptAppTests/Functionality/Mail Provider/GmailServiceTest.swift

```diff
@@ -46,7 +46,7 @@ class GmailServiceTest: XCTestCase {
 class GoogleUserServiceMock: GoogleUserServiceType {
     var authorization: GTMAppAuthFetcherAuthorization?
     func renewSession() async throws {
-        await Task.sleep(1_000_000_000)
+        try await Task.sleep(nanoseconds: 1_000_000_000)
```

Comment:
I was earlier wondering what is the difference between the two

Reply:
Xcode showed deprecation warning for `await Task.sleep(1_000_000_000)`, so it's the same, only named parameter `nanoseconds` was added